swagger_to_sdk_config.json

```diff
@@ -6,18 +6,18 @@
       "dep ensure",
       "go install",
       "cd ../..",
-      "go generate",
+      "go generate ./profiles/...",
       "gofmt -w ./services/",
       "gofmt -w ./profiles/"
     ],
     "autorest_options": {
-      "use": "@microsoft.azure/autorest.go@preview",
+      "use": "@microsoft.azure/autorest.go@~2",
```

Comment:
Not sure if autorest just calls npm install on this, but last minor version of autorest.go is 2.1.x .  Usually these versions are specified as ~2.1.0 , not sure what is the expected behavior on ~2

Reply:
~2 ties it to the major version, as you propose above :)

Reply:
ok apparently just doing ~2 ties it to the major version just that standard node practice is doing ^2.1.0 but if this works I guess its fine.